docker-compose.yml

```diff
@@ -4,19 +4,12 @@ services:
   stafl-devcontainer:
     image: "ghcr.io/staflsystems/stafl-devcontainer-ci:latest"
     command: sleep infinity
-    network_mode: service:kroki
     volumes:
       - ..:/workspaces:cached
       - data:/home/vscode
   kroki:
-    image: yuzutech/kroki:latest
-    ports:
-      - 8000:8000
+    image: ghcr.io/staflsystems/kroki:latest
```

Comment:
No need to map ports?

Reply:
nope, it's only accessed from within the compose network